.travis.yml

```diff
@@ -27,3 +34,10 @@ script:
   - git clone --depth 1 https://github.com/regro/cf-graph3
   - cd cf-graph3
   - python -m conda_forge_tick.test_status
+  - set -e
+  - cd ..
+  - cd docs
+  - make html
+  - cd ..
+  - doctr deploy . --built-docs ./docs/_build/html;
```

Comment:
Shouldn't this go to the gh-pages branch?

Reply:
I think it does?
from the help
```
--deploy-branch-name DEPLOY_BRANCH_NAME
                        Name of the branch to deploy to (default: 'master' for
                        ``*.github.io`` and wiki repos, 'gh-pages' otherwise)
```

Reply:
Ok cool